pkg/auth/scope/publicshare.go

```diff
@@ -35,6 +34,9 @@ import (
 	"github.com/rs/zerolog"
 )
 
+// PublicStorageProviderID is the space id used for the public links storage space
+const PublicStorageProviderID = "7993447f-687f-490d-875c-ac95e89a62a4"
```

Comment:
this is very dangerous. It should live in the config.go default structs instead of here, or we are setting ourselves up to failures in the future.

Reply:
Moved the declaration to `utils` pkg in https://github.com/cs3org/reva/pull/2351

I don't think it makes much sense to have a uuid configurable. If you (for whatever reason) need to change it you can still overwrite it via build flags